tests/foreman/ui/test_errata.py

```diff
@@ -163,37 +173,89 @@ def vm(module_repos_collection_with_setup, rhel7_contenthost, target_sat):
     return rhel7_contenthost
 
 
+@pytest.fixture(scope='function')
+def registered_contenthost(
+    rhel_contenthost,
+    module_org,
+    module_lce,
+    module_target_sat,
+    repos=[CUSTOM_REPO_URL],
+):
+    """RHEL ContentHost registered in satellite,
+    Using SCA and global registration.
+
+    :param repos: list of upstream URLs for custom repositories,
+        default to CUSTOM_REPO_URL
+    """
+    content_view = module_target_sat.api.ContentView(
+        organization=module_org,
+        environment=[module_lce],
+    ).create()
+
+    activation_key = module_target_sat.api.ActivationKey(
+        organization=module_org,
+        environment=module_lce,
+    ).create()
+
+    custom_repos = []
+    for repo_url in repos:
+        rhel_contenthost.create_custom_repos(custom_repo=repo_url)
+        # Publishes and promotes a new cvv to lce
+        # Associate org, ak, cv, with custom repo:
+        custom_repo_id = module_target_sat.cli_factory.setup_org_for_a_custom_repo(
+            {
+                'url': repo_url,
+                'organization-id': module_org.id,
+                'lifecycle-environment-id': module_lce.id,
+                'activationkey-id': activation_key.id,
+                'content-view-id': content_view.id,
+            }
+        )['repository-id']
+        custom_repos.append(custom_repo_id)
+
+    result = rhel_contenthost.register(
+        activation_keys=activation_key.name,
+        lifecycle_environment=module_lce,
+        target=module_target_sat,
+        org=module_org,
+        loc=None,
+    )
+    assert result.status == 0, f'Failed to register host:\n{result.stdout}'
```

Comment:
is stdout the most appropriate to use here or would stderr be better for the error message?

Reply:
right, the full output is probably unnecessary here, I'll push a change to `result.stderr`